SPECS/initramfs/initramfs.spec

```diff
@@ -93,10 +93,9 @@ if [ -f %{_localstatedir}/lib/rpm-state/initramfs/regenerate ]; then \
 elif [ -d %{_localstatedir}/lib/rpm-state/initramfs/pending ]; then \
     for k in `ls %{_localstatedir}/lib/rpm-state/initramfs/pending/`; do \
         echo "(re)generate initramfs for $k," %* >&2 \
+        mkinitrd -q /boot/initrd.img-$k $k -k \
         if [[ $k == *mshv* ]]; then \
-            mkinitrd -q /boot/efi/initrd.img-$k $k -k \
-        else \
-            mkinitrd -q /boot/initrd.img-$k $k -k \
+            cp /boot/initrd.img-$k /boot/efi/initrd.img-$k \
```

Comment:
Would a symlink be possible here to reduce disk usage? Initrds aren't massive, but they aren't small either.

Reply:
Tried this; symlinks don't work across device boundaries: 

![image](https://github.com/microsoft/CBL-Mariner/assets/37827103/b897a87d-c8c4-4012-936e-891b623c5f93)